app/(you)/(edit-profile)/edit-profile-last-name.tsx

```diff
@@ -0,0 +1,92 @@
+import { router, useNavigation } from 'expo-router';
+import React, { useContext, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { View } from 'react-native';
+
+import Box from '@/components/ui/box/Box';
+import CustomHeader from '@/components/ui/header/CustomHeader';
+import CustomImage from '@/components/ui/image/CustomImage';
+import MultiTextInput from '@/components/ui/text-input/MultiTextInput';
+import { ProfileUpdateContext } from '@/context/userdetails/ProfileUpdateContext';
+import { Validator } from '@/util/Validator';
+
+export default function EditProfileLastName() {
+  const { t } = useTranslation();
+
+  const { profileUpdate, setProfileUpdate } = useContext(ProfileUpdateContext);
+
+  const validator = new Validator([
+    {
+      rule(arg: string) {
+        return arg.length >= 2;
+      },
+      errorMessage: t('Last name must contain at least 2 characters'),
+    },
+    {
+      rule(arg: string) {
+        return arg.length <= 20;
+      },
+      errorMessage: t('Last name may contain at most 20 characters'),
```

Comment:
nie wiem czy to jest walidne w przypadku niektórzych chinczyków,ale póki co chyba nie musimy się tym bujać